packages/components/src/List/List.component.js

```diff
@@ -95,26 +103,45 @@ ListToolbar.propTypes = {
 }
  <List {...props}></List>
  */
-function List({ displayMode, id, list, toolbar, defaultHeight, renderers, rowHeight }) {
+function List({
+	displayMode,
+	id,
+	list,
+	toolbar,
+	defaultHeight,
+	rowHeight,
+	getComponent,
+	components = {},
+}) {
 	const classnames = classNames('tc-list', theme.list);
+	const injected = Inject.all(getComponent, omit(components, ['toolbar', 'list']));
+
 	return (
 		<div className={classnames}>
+			{injected('before-toolbar')}
```

Comment:
`{injected('before-list-wrap')}` ? Could be nice for something like `Breadcrumb`

Reply:
yes it's for customization !! or tabs for example